Sprint-2/implement/bmi.js

```diff
@@ -13,3 +13,24 @@
 // Given someone's weight in kg and height in metres
 // Then when we call this function with the weight and height
 // It should return their Body Mass Index to 1 decimal place
+
+
+function ibmCalculator() {
```

Comment:
What would happen if you had a different height or weight?

Reply:
I understand now why it was giving the same results. I removed the constant from inside the function and fixed the height units.

Reply:
Fix the units/ removed the constant values